src/baseComponents/progressBar/ProgressBar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { CircularProgress } from "material-ui";
+import testUtils from "../../testUtils/utils";
+
+export default class ProgressBar extends React.Component {
+  render() {
+    const mode = testUtils.isRunningTests() ? "determinate" : "indeterminate";
```

Comment:
We need to disable animations if we are going to use screenshot testing :) I'd like to use a dependency injector, but we don't have one for this app.